src/components/pages/Favorites/Component.js

```diff
@@ -18,10 +33,14 @@ function favtitle() {
       }}
     >
       {' '}
-      Your favorite cities are saved on the left. Select up to three and see how
-      they stack up against each other!
+      View your favorite cities 🏙️
+      <div>
+        {dummyFavorites.map(city => {
```

Comment:
I think it's good that you named this 'dummyData' as we know what to remove when we go to production and get 'real' data incoming!